sources/packages/api/src/route-controllers/student/student.students.controller.ts

```diff
@@ -199,4 +200,15 @@ export class StudentStudentsController extends BaseController {
       fileMetadata,
     );
   }
+
+  /**
+   * Get the student information that represents the profile.
+   * @returns student profile information.
+   */
+  @Get()
+  async getStudentProfile(
+    @UserToken() userToken: StudentUserToken,
+  ): Promise<StudentProfileAPIOutDTO> {
+    return this.studentControllerService.getStudentProfile(userToken.studentId);
```

Comment:
The studentId in user token has made things so easy ❤️

Reply:
Agree, it is coming in handy in many controllers 👍